scripts/alluxio/alluxio-systemd.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
the things in scripts aren't really end user scripts right now.
We are also adding docker image here: https://github.com/NVIDIA/spark-rapids/pull/6485

And I would expect that to use these scripts as well.  @NVnavkumar 
I think I would be more in favor of putting these under the docker/databricks/scripts directory for now and the docs can link to it there.  If we start supporting alluxio out of the box on other platforms we can discuss moving.

Reply:
Will add it to docker image